Translations of `Translations` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Translations](https://hosted.weblate.org/projects/osrd/home/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/home/horizontal-auto.svg)

gaetan-osrd commented on chart/templates/stateful_editoast/deployment.yaml:
```diff
@@ -13,7 +13,7 @@ metadata:
     {{- toYaml $mergedAnnotations | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.services.statefulEditoast.replicas | default 2 }}
```
Change unrelated to the PR

Commit:
front: home: add translation using weblate (Chinese (Simplified Han script))

Signed-off-by: Languages add-on <noreply-addon-languages@weblate.org>

wf16110144 committed:
front: home: translated using weblate (Chinese (Simplified Han script))

Currently translated at 0.1% (1 of 631 strings)

Translation: OSRD/Translations
Translate-URL: https://hosted.weblate.org/projects/osrd/home/zh_Hans/

Signed-off-by: 王飞 <1071607129@qq.com>

flomonster committed:
Deleted translation using Weblate (Chinese (Simplified Han script))

flomonster commented:
Empty PR
